Package.swift

```diff
@@ -9,9 +9,9 @@ let package = Package(
         .Package(url: "https://github.com/vapor-community/cpostgresql.git", beta),
```

Comment:
Let's update cpostgresql line to `majorVersion: 2` as well. We can merge in this PR when the CPostgreSQL library is released as 2.0.
Remove the `beta` variable above as well. 

Thanks!